### DISCUSSION TITLE
Add Personal links

### DISCUSSION THREAD

**Discussion by Coderproz** [dailydotdev/daily#868] [Feature Requests]
I have many sites that I visit frequently and I want them to stay in front of me and not in my bookmarks. If it is possible to add such an option, it may be in the list on the left, as well as the YouTube menu.

Thank you all.

**Comment by idoshamun** [comment]
Hi @Coderproz,

We have a feature called custom shortcuts exactly for that.
Read more here: https://docs.daily.dev/docs/getting-started/browser-extension-installation#adding-custom-shortcuts-to-dailydev
